xbmc/addons/AddonManager.cpp

```diff
@@ -1025,4 +1050,41 @@ void CAddonMgr::FindAddons(ADDON_INFO_LIST& addonmap, const std::string& path)
   }
 }
 
+bool CAddonMgr::IsFromTrustedRepo(const AddonPtr& addon) const
+{
+  for (const auto& trustedRepo : m_trustedAddonRepos)
```

Comment:
Sidenote: You can take a look at `std::any_of` if you're bored (it's fine like this though)

Reply:
will have a look at it :)